src/main.ts

```diff
@@ -1,46 +1,101 @@
-import { Client, LocalAuth } from "whatsapp-web.js";
+import { Client, LocalAuth } from 'whatsapp-web.js';
 
 const groupId = process.env.GROUP_ID;
 
 const client = new Client({
   authStrategy: new LocalAuth(),
   puppeteer: {
-    args: ["--no-sandbox"],
+    args: ['--no-sandbox'],
   },
 });
 
-client.on("qr", (qr) => {
+const nettoInsults = [
+  'sim netto é boiola gosta de chupar pika e quicar num cacete grande e ficar rebolando quando vê homem sarado ✨',
+  'siiiiiim netinho muito gay pra mim ✨',
+  'sim netto gosta de dançar de salto alto e fazer carão no espelho 💅',
+  'sim netto tem um crush no Superman e sonha em ser salvo por ele 🦸‍♂️',
+  'nossa, netinho gay pra mim ✨',
+  'sim netto é apaixonado por unicórnios e coleciona todos os produtos relacionados 🦄',
+  'sim netto é usuario de Crocs 👟',
+  'sim netto tem elo WhatsApp fake no Valorant 😂',
+  'Netto é muito domador de piroca',
+  'sim netto é um verdadeiro mestre no Valorant, todos os inimigos tremem diante dele 🔫',
+  'netinho gay pra mim ✨',
+  'Salve',
+  'Egirl e amor',
+  'egirl e vida',
+  'melhor assunto do mundo',
+  'Egirl',
+  'lindas',
+  'goticas',
+  'KKKKKKKKKKK',
+];
+
+const learning: string[] = [...nettoInsults, 'Remulo tambem é gay ✨'];
+
+client.on('qr', (qr) => {
   console.log(qr);
 });
 
-client.on("ready", () => {
-  console.log("Client is ready!");
+client.on('ready', () => {
+  console.log('Client is ready!');
 });
 
-client.on("message", async (msg) => {
-  if (msg.from !== groupId) {
+client.on('message', async (msg) => {
+  if (msg.from === groupId) {
```

Comment:
Guard do grupo deve estar no começo da função e retornar se o groupId for diferente